### PR TITLE
Allow more ES endpoints

### PR DESCRIPTION
**Pull request by aolieman**
I'd like to use some (safe) ES endpoints that haven't been enabled yet.

To start, I've added:
- `GET /_cat/indices/{index}`
- `GET /{index}/_doc/{docId}`
- `GET /{index}/_source/{docId}`

Maintainers can edit this feature branch, so feel free to fix any mistakes you might find.

### CODE REVIEW

**Comment by jurrian** [comment]
Please allow edits from maintainers:
https://help.github.com/en/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork

**Comment by aolieman** [comment]
image
 
I've allowed edits from maintainers when I first opened this PR. If you're not able to push to `WaarOverheid:feature/more-es-endpoints`, that might have something to do with being an organization repo. I can give you access to the downstream repo if you can't push to the PR branch already.

File: deployment/endpoints/staging.yaml [hunk 1]
```diff
@@ -41,10 +41,24 @@ paths:
...
           description: ""
 
+  /_cat/indices/{index}:
+    get:
+      tags:
+        - "Elasticsearch API"
+      description: |
+        Elasticsearch endpoint displaying the indices that match the index filter.
+        See https://www.elastic.co/guide/en/elasticsearch/reference/current/cat-indices.html for more information.
+      operationId: "elastic_indices_get_index"
```
Comment (jurrian) [inline deployment/endpoints/staging.yaml]:
```suggestion
      operationId: "elastic_indices_index_get"
```

File: deployment/endpoints/staging.yaml [hunk 2]
```diff
@@ -149,10 +163,46 @@ paths:
...
           description: ""
 
+  /{index}/_doc/{docId}:
+    get:
+      tags:
+        - "Elasticsearch API"
+      description: |
+        Elasticsearch endpoint for retrieving a single document.
+        See https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-get.html for more information.
+      operationId: "elastic_doc_get"
```
Comment (jurrian) [inline deployment/endpoints/staging.yaml]:
```suggestion
      operationId: "elastic_doc_index_get"
```

File: deployment/endpoints/staging.yaml [hunk 3]
```diff
@@ -149,10 +163,46 @@ paths:
...
+          description: ""
+
+  /{index}/_source/{docId}:
+    get:
+      tags:
+        - "Elasticsearch API"
+      description: |
+        Elasticsearch endpoint for retrieving a single document source.
+        See https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-get.html for more information.
+      operationId: "elastic_source_get"
```
Comment (jurrian) [inline deployment/endpoints/staging.yaml]:
```suggestion
      operationId: "elastic_source_index_get"
```

File: deployment/endpoints/production.yaml [hunk 4]
```diff
@@ -41,10 +41,24 @@ paths:
...
           description: ""
 
+  /_cat/indices/{index}:
+    get:
+      tags:
+        - "Elasticsearch API"
+      description: |
+        Elasticsearch endpoint displaying the indices that match the index filter.
+        See https://www.elastic.co/guide/en/elasticsearch/reference/current/cat-indices.html for more information.
+      operationId: "elastic_indices_get_index"
```
Comment (jurrian) [inline deployment/endpoints/production.yaml]:
```suggestion
      operationId: "elastic_indices_index_get"
```

File: deployment/endpoints/production.yaml [hunk 5]
```diff
@@ -149,10 +163,46 @@ paths:
...
           description: ""
 
+  /{index}/_doc/{docId}:
+    get:
+      tags:
+        - "Elasticsearch API"
+      description: |
+        Elasticsearch endpoint for retrieving a single document.
+        See https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-get.html for more information.
+      operationId: "elastic_doc_get"
```
Comment (jurrian) [inline deployment/endpoints/production.yaml]:
```suggestion
      operationId: "elastic_doc_index_get"
```

File: deployment/endpoints/production.yaml [hunk 6]
```diff
@@ -149,10 +163,46 @@ paths:
...
+          description: ""
+
+  /{index}/_source/{docId}:
+    get:
+      tags:
+        - "Elasticsearch API"
+      description: |
+        Elasticsearch endpoint for retrieving a single document source.
+        See https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-get.html for more information.
+      operationId: "elastic_source_get"
```
Comment (jurrian) [inline deployment/endpoints/production.yaml]:
```suggestion
      operationId: "elastic_source_index_get"
```

**Comment by aolieman** [comment]
Looks good to me!
I'm currently in the bush with dodgy connection. Do you need anything from me?

**Comment by jurrian** [comment]
I can't make the necessary changes because of access rights, and since @aolieman is abroad I have created a new PR #268.

**Pull request closed without merging**